vllm/logger.py

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 """Logging configuration for vLLM."""
+from __future__ import annotations
+
```

Comment:
Is trhis used?

Reply:
ah I need to turn off this in my editor, it automatically add it to files that doesn't have it.